src/Infrastructure/LeanCode.ForceUpdate/CQRSServicesBuilderCQRSExtensions.cs

```diff
@@ -0,0 +1,20 @@
+using LeanCode.Components;
+using LeanCode.CQRS.AspNetCore;
+using LeanCode.ForceUpdate.Contracts;
+using LeanCode.ForceUpdate.Services.CQRS;
+using LeanCode.ForceUpdate.Services;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace LeanCode.ForceUpdate;
+
+public static class CQRSServicesBuilderCQRSExtensions
+{
+    public static CQRSServicesBuilder AddForceUpdate(this CQRSServicesBuilder cqrsServicesBuilder)
+    {
+        cqrsServicesBuilder.services.AddTransient<VersionHandler>();
```

Comment:
Since `VersionSupportQH` will throw if you don't register version configurations and I don't see any way to dynamically reload support versions now, maybe let's require versions to be passed explicitly here and register them as singletons?